triplestore-jena-memory/src/main/java/zone/cogni/asquare/triplestore/pool/key/ConceptUriBasedPoolKey.java

```diff
@@ -0,0 +1,57 @@
+package zone.cogni.asquare.triplestore.pool.key;
+
+import org.springframework.util.StringUtils;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Objects;
+
+public class ConceptUriBasedPoolKey implements RdfStoreServicePoolKey<String> {
+
+  private final Path dirPath;
+
+  public ConceptUriBasedPoolKey(final Path basePath, final String conceptUri) {
```

Comment:
The idea was to share the URI to "dir name" functionality with a base implementation and then each "real" implementation can inherit later from here. (because each real key needs to contain all the information what is needed to create a pooled rdf store service)
What would you suggest as a change?

Reply:
Like I mentioned. I am not 100% sure what we are trying to achieve.  The parameter conceptUri is a bit confusing. How is it a conceptUri?

Reply:
what would you suggest to change it? merge the two classes?